catalog/core/catalog-core-solr/src/test/java/ddf/catalog/source/solr/DynamicSchemaResolverTest.java

```diff
@@ -97,6 +98,63 @@ public void testAddFields() throws Exception {
     }
   }
 
+  /**
+   * Verify that when the metadata size limit is set to 0 (less than the size of what is being added
+   * to the SolrInputDocument) that it is not added to the SolrInputDocument
+   */
+  @Test
+  public void testAddFieldsDoesntAddMetadataSizeLimitIsZero() throws Exception {
+    // Set
+    System.setProperty("metadata.size.limit", "0");
```

Comment:
❓Since `0` could be a special case, should we set the limit to something like `1` or `10` and have the `mockAttribute.getValue()` return something larger than that to make sure the limit is properly enforced?

Reply:
changed to 10. fixed